build.gradle

```diff
@@ -2,13 +2,11 @@
 
 buildscript {
     repositories {
+        google()
```

Comment:
This needs to be fixed in a few places I think.

Reply:
One more in the same file, true. I have pushed an updated version of `build.gradle`.